frida_tools/lsd.py

```diff
@@ -19,9 +19,20 @@ def _start(self) -> None:
                 self._update_status(f"Failed to enumerate devices: {e}")
                 self._exit(1)
                 return
+            device_os = {}
+            for device in devices:
+                try:
+                    params = device.query_system_parameters()
+                    if "name" in params:
+                        device.name = params["name"]
+                    if "os" in params:
+                        device_os[device.id] = params["os"]["name"] + " " + params["os"]["version"]
```

Comment:
```suggestion
                        device_os[device.id] = params["os"]["name"] + " " + params["os"]["version"]
```

Need a check for `version` as it's not required. Would probably also be good to extract `params["os"]` to a local variable while at it.